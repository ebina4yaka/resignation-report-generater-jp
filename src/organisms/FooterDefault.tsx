import React from 'react'
import Typography from '@material-ui/core/Typography'
import makeStyles from '@material-ui/core/styles/makeStyles'
import Copyright from '../atoms/Copyright'

const useStyles = makeStyles((theme) => ({
  root: {
    backgroundColor: theme.palette.background.paper,
    padding: theme.spacing(6),
  },
}))

export default function FooterDefault(): React.ReactElement {
  const classes = useStyles()
  return (
    <footer className={classes.root}>
      <Typography variant="h6" align="center" gutterBottom>
        退職届ジェネレーター
      </Typography>
      <Typography
        variant="subtitle1"
        align="center"
        color="textSecondary"
        component="p"
      >
        Released under the MIT License.
      </Typography>
      <Copyright />
    </footer>
  )
}
