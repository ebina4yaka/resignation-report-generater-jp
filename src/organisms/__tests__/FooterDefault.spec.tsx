import renderer from 'react-test-renderer'
import FooterDefault from '../FooterDefault'

jest.mock('../../atoms/Copyright', () => 'MockedCopyright')
jest.mock('next/link', () => 'MockedLink')

test('Snapshot test for FooterDefault', () => {
  const tree = renderer.create(<FooterDefault />).toJSON()
  expect(tree).toMatchSnapshot()
})
