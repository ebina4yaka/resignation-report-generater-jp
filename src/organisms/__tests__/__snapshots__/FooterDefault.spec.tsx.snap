// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for FooterDefault 1`] = `
<footer
  className="makeStyles-root-1"
>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter"
  >
    退職届ジェネレーター
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom MuiTypography-alignCenter"
  >
    <MockedLink
      href="/"
      passHref={true}
    >
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        トップページ
      </a>
    </MockedLink>
  </p>
  <p
    className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Released under the MIT License.
  </p>
  <MockedCopyright />
</footer>
`;
